src/components/events-list/warning-state.js

```diff
@@ -1,27 +1,29 @@
-import { faCheckCircle } from '@fortawesome/free-solid-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import Chip from '../chip';
+import JobStatusChip from '../job-status-chip';
+import { Icon } from '@equinor/eds-core-react';
+import { check, warning_outlined } from '@equinor/eds-icons';
 
 import eventModel from '../../models/event';
 import { isEventObsolete, isEventResolved } from '../../utils/event-model';
 
 const WarningResolved = () => {
   return (
     <React.Fragment>
-      <Chip type="info">
-        <FontAwesomeIcon icon={faCheckCircle} /> Resolved
-      </Chip>
+      <JobStatusChip type="success">
```

Comment:
use 'type={ jobStatus.SUCCESS }' instead of adding your own icon here as this will give you the same effect